kahuna/public/js/components/gr-collections-panel/gr-collections-panel.js

```diff
@@ -114,7 +114,9 @@ grCollectionsPanel.controller('GrNodeCtrl',
     };
 
     subscribe$($scope, pathWithImages$, ({path, images}) => {
-       collections.addImagesToCollection(images, path).then(() => ctrl.saving = false);
+       collections.addImagesToCollection(images, path).then(() => {
+           $scope.$apply(() => ctrl.saving = false);
```

Comment:
I don't understand why we need an `$apply` here, as `$http` and `$q` are supposed to do that for you?